lib/roo_on_rails/railties/sidekiq.rb

```diff
@@ -11,18 +11,18 @@ class Sidekiq < Rails::Railtie
         require 'hirefire-resource'
         
         if RooOnRails::Config.sidekiq_enabled?
-          $stderr.puts 'initializer roo_on_rails.sidekiq'
+          Rails.logger.debug 'initializer roo_on_rails.sidekiq'
           config_sidekiq
           config_sidekiq_metrics
           config_hirefire(app)
         else
-          $stderr.puts 'skipping initializer roo_on_rails.sidekiq'
+          Rails.logger.debug 'skipping initializer roo_on_rails.sidekiq'
         end
       end
 
       def config_hirefire(app)
         unless ENV['HIREFIRE_TOKEN']
-          warn 'No HIREFIRE_TOKEN token set, auto scaling not enabled'
+          Rails.logger.warn 'No HIREFIRE_TOKEN token set, auto scaling not enabled'
```

Comment:
Might be nice to include `initializer: 'roo_on_rails.sidekiq` as part of the metadata for these messages; you could use a `with` block around them to do it.

Reply:
Done - this call is wrapped inside a `Logger#with` block now.